qualtran/_infra/registers_test.py

```diff
@@ -135,6 +143,13 @@ def test_signature_symbolic():
     assert str(signature.n_qubits()) == 'n_x + n_y'
 
 
+def test_partial_classical_signature_n_qubits():
+    sig = Signature(
+        [Register('x', QUInt(5)), Register('y', QUInt(5), side=Side.RIGHT), Register('c', CBit())]
+    )
+    assert sig.n_qubits() == 10
```

Comment:
Also assert for `sig.n_cbits()` (and if doesn't exist?)

Reply:
added `sig.n_cbits()` and `sig.n_bits()` for consistency with `Register` and `QCDType` and asserted them here; thanks. 

I kept the spellings consistent _within_ the class, so it's  `QCDType.num_xbits`, `Register.total_xbits`, `Signature.n_xbits`. Maybe we can provide a consistent spelling (in addition to the historical one(s)) in a follow-on PR.